packages/design-system/src/styles/components/_Choice.scss

```diff
@@ -13,11 +13,12 @@ $ds-c-inset-border-width: $spacer-half;
 
   // Choice label
   + label {
-    align-items: center;
-    cursor: pointer;
+    align-items: flex-start;
     display: flex;
+    flex-direction: column;
     flex-wrap: wrap;
     font-weight: $font-normal;
+    justify-content: center;
     margin: $spacer-1 0;
     max-width: $measure-base;
```

Comment:
```suggestion
    max-width: $measure-base;
    width: fit-content;
```

Reply:
Thanks - it's ready for a rerun @bernardwang @line47